src/index.html

```diff
@@ -1,12 +1,318 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="nav__link is-active" href="#">Apple</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Samsung</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Smartphones</a></li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#" data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item"><a class="nav__link" href="#">Gadgets</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">Tablets</a></li>
```

Comment:
Add blank line between the links for better readability
```suggestion
          <li class="nav__item">
            <a class="nav__link" href="#">Gadgets</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#">Tablets</a>
          </li>
```